packages/build-tools/src/steps/functions/installPods.ts

```diff
@@ -0,0 +1,18 @@
+import { BuildFunction } from '@expo/steps';
+import spawn from '@expo/turtle-spawn';
+
+export function createInstallPodsBuildFunction(): BuildFunction {
+  return new BuildFunction({
+    namespace: 'eas',
+    id: 'install_pods',
+    name: 'Install Pods',
+    fn: async (stepsCtx, { env }) => {
+      stepsCtx.logger.info('Installing pods');
+      await spawn('pod', ['install'], {
+        stdio: 'pipe',
+        env,
+        cwd: stepsCtx.workingDirectory,
```

Comment:
Shouldn't this execute in `ios` or something? What about monorepos?

Reply:
It should

Reply:
What I meant by mentioning monorepos is that if I have a monorepo where the Expo app is under `packages/mobile-app` or something, things will go bad.

I think it's ok though for v1, as you suggest. Let's let users build normal apps with `eas/build` and let the more advanced ones use specific steps from examples.

Reply:
If we do working directory = `./ios` it will be run in the correct directory because after the `checkout` the `.` resolves to react native project directory

Reply:
So it should also work for monorepos

Reply:
Fixing it